CMakeLists.txt

```diff
@@ -79,6 +86,11 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL iOS)
 	set(suffix_platform _ios)
 elseif(CMAKE_SYSTEM_NAME STREQUAL Android)
 	set(suffix_platform _android)
+elseif(CMAKE_SYSTEM_NAME STREQUAL Emscripten)
+	set(CMAKE_SHARED_LIBRARY_CREATE_C_FLAGS "-sSIDE_MODULE=1")
+	set(CMAKE_SHARED_LIBRARY_CREATE_CXX_FLAGS "-sSIDE_MODULE=1")
+	set(CMAKE_SHARED_LIBRARY_SUFFIX ".wasm")
```

Comment:
I would have thought CMake would do this automatically? What's the default suffix?

Reply:
The default is .so and chromium was refusing to load them until I renamed to .wasm. (Godot does this too.)